src/app/pages/RoflAppDetailsPage/index.tsx

```diff
@@ -96,124 +105,224 @@ export const StyledGrid = styled(Grid)(({ theme }) => ({
+      <TeeRow policy={app.policy} />
+      <DetailsRow title={t('rofl.appId')}>
+        <Typography variant="mono" component="span">
+          {app.id}
+        </Typography>
+        <CopyToClipboard value={app.id} />
+      </DetailsRow>
+      <DetailsRow title={t('rofl.enclaveId')}>
+        <Enclaves policy={app.policy} />
+      </DetailsRow>
+      <DetailsRow title={t('rofl.sekPublicKey')}>
+        <Typography variant="mono">{app.sek}</Typography> <CopyToClipboard value={app.sek} />
+      </DetailsRow>
+      <AdminAccountRow
+        address={app.admin_eth ?? app.admin}
+        scope={{ network: app.network, layer: app.layer }}
+      />
+      <StakedAmountRow stake={app.stake} ticker={app.ticker} />
+      <StatusBadgeRow hasActiveInstances={!!app.num_active_instances} removed={app.removed} />
+      <ActiveInstancesNumberRow number={app.num_active_instances} />
+      <LastActivityRow
+        scope={{ network: app.network, layer: app.layer }}
+        transaction={app.last_activity_tx}
+      />
+      <DetailsRow title={t('rofl.endorsement')}>
+        <Endorsement policy={app.policy} />
+      </DetailsRow>
+      <DetailsRow title={t('rofl.secrets')}>
+        <Secrets secrets={app.secrets} />
+      </DetailsRow>
+    </StyledDescriptionList>
+  )
+}
+
+export const RoflAppDetailsViewSearchResult: FC<{
+  isLoading?: boolean
+  app: RoflApp | undefined
+  highlightedPartOfName?: string
+}> = ({ app, isLoading, highlightedPartOfName }) => {
+  const { t } = useTranslation()
+  const { isMobile } = useScreenSize()
+
+  if (isLoading) return <TextSkeleton numberOfRows={10} />
+  if (!app) return <></>
+
+  return (
+    <StyledDescriptionList titleWidth={isMobile ? '100px' : '200px'}>
+      <DetailsRow title={t('common.paratime')}>
+        <DashboardLink scope={{ network: app.network, layer: app.layer }} />
+      </DetailsRow>
+      <NameRow name={app.metadata['net.oasis.rofl.name']} />
+      <VersionRow version={app.metadata['net.oasis.rofl.version']} />
+      <TeeRow policy={app.policy} />
+      <DetailsRow title={t('rofl.appId')}>
+        <RoflAppLink
+          id={app.id}
+          name={app.id}
+          network={app.network}
+          highlightedPartOfName={highlightedPartOfName}
+          withSourceIndicator={false}
+        />
+        <CopyToClipboard value={app.id} />
+      </DetailsRow>
+      <AdminAccountRow
+        address={app.admin_eth ?? app.admin}
+        scope={{ network: app.network, layer: app.layer }}
+      />
+      <StakedAmountRow stake={app.stake} ticker={app.ticker} />
+      <StatusBadgeRow hasActiveInstances={!!app.num_active_instances} removed={app.removed} />
+      <ActiveInstancesNumberRow number={app.num_active_instances} />
+      <LastActivityRow
+        scope={{ network: app.network, layer: app.layer }}
+        transaction={app.last_activity_tx}
+      />
     </StyledDescriptionList>
   )
 }
+
+export const RoflAppDetailsVerticalListView: FC<{
```

Comment:
Are we doing single component exports per file? This can affect HMR.

Reply:
good point, will validate this

Reply:
HMR works just fine